dynawo/sources/Models/CPP/ModelOmegaRef/DYNModelOmegaRef.cpp

```diff
@@ -109,8 +108,7 @@ ModelOmegaRef::init(const double& /*t0*/) {
   nbCC_ = 0;
 
   ModelOmegaRef::col1stOmegaRef_ = 0;
-  ModelOmegaRef::col1stTetaRef_ = ModelOmegaRef::col1stOmegaRef_ + nbMaxCC;
-  ModelOmegaRef::col1stOmega_ = ModelOmegaRef::col1stTetaRef_ + nbMaxCC;
+  ModelOmegaRef::col1stOmega_ = ModelOmegaRef::col1stOmegaRef_ + nbMaxCC;
   ModelOmegaRef::col1stOmegaRefGrp_ = ModelOmegaRef::col1stOmega_ + nbOmega_;
```

Comment:
Not related to your commit: why are we using a static member here whereas the value is dependent on a local member?

Reply:
Done